components/automate-cli/cmd/chef-automate/certRotate.go

```diff
@@ -510,6 +512,10 @@ func (c *certRotateFlow) getCerts(infra *AutomteHAInfraDetails, flagsObj *flags)
 			fmt.Sprintf(fileAccessErrorMsg, err.Error()),
 		)
 	}
+	block, _ := pem.Decode(privateCert)
+	if block == nil {
+		return &certificates{}, errors.New("Please provide a valid certificate")
```

Comment:
instead of a general statement, you can write `Please provide the valid certificate for privateCert`
similar statements for other changes as well